airflow/providers/amazon/aws/waiters/emr-serverless.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": 2,
+    "waiters": {
+        "no_job_running": {
+            "operation": "ListJobRuns",
+            "delay": 10,
+            "maxAttempts": 60,
+            "acceptors": [
+                {
+                    "matcher": "path",
+                    "argument": "length(jobRuns) == `0`",
+                    "expected": true,
+                    "state": "success"
+                }
+            ]
```

Comment:
I don't like too much the fact that I don't have a failure case for this waiter, but I think there is nothing we can do about it...
The failure case would be count > prev_count, or detecting a new job_id that we didn't see in the previous calls, which is way beyond the capabilities of waiters.